### ISSUE TITLE
[Accessibility]: No easy way to exit autoindex job logs and move onto next 

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35264]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513, https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

~14:45 in the recording, when in the logs, we have to go through them line-by-line. The next heading is all the way at the bottom, at "This job uploaded an index X minutes ago", skipping all the remaining timeline steps.

### Expected behavior

Each step is a heading, making navigating between each step more convenient and quicker.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
I don't think this is an issue anymore, as the logs seem to be collapsed by default, so you can skip over them by default and need to manually expand them.
